web/components/templates/prompts/experiments/scoresTable.tsx

```diff
@@ -0,0 +1,197 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeaderCell,
+  TableRow,
+} from "@tremor/react";
+import ModelPill from "../../requestsV2/modelPill";
+import { clsx } from "../../../shared/clsx";
+
+type ScoreValue = string | number;
+
+type ExperimentScores = {
+  dataset: {
+    scores: Record<string, ScoreValue>;
+  };
+  hypothesis: {
+    scores: Record<string, ScoreValue>;
+  };
+};
+
+export type ScoresProps = {
+  scores: ExperimentScores;
+};
+const ScoresTable = ({ scores }: ScoresProps) => {
+  const getScoreValue = (score: ScoreValue, field: string) => {
+    if (field === "dateCreated" && typeof score === "string") {
+      return renderScoreValue(score);
+    }
+    if (field === "cost" && typeof score === "number") {
+      return `$${score.toFixed(4)}`;
+    }
+    if (field === "model" && typeof score === "string") {
+      return <ModelPill model={score} />;
+    }
+    return (score as any)[field];
```

Comment:
nit: we should probably remove this `as any` cast or make it safer. I can imagine this causing some application error down the road. maybe wrap it in a try catch and see if you can, otherwise, default to null or empty string